external/pybind11.cmake

```diff
@@ -24,7 +24,8 @@
 if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/pybind11-2.2.4.tar.gz)
     set(URL ${REPOSITORY_DIR}/build/ThirdParty/share/pybind11-2.2.4.tar.gz)
 else()
-    set(URL https://github.com/pybind/pybind11/archive/v2.2.4.tar.gz)
+    #set(URL https://github.com/pybind/pybind11/archive/v2.2.4.tar.gz)
+    set(URL https://github.com/pybind/pybind11/archive/master.zip)
```

Comment:
why do we need master?

Reply:
> why do we need master?

I was trying to get Python 3.7 to work.  But using master does not do it. When I run Python 3.7 it complains about the Python36.dll not being compatible and will not load the extension libraries.

I will revert this back to the .tar.gz download and wait for the next version of pybind11.

Reply:
In most cases we want to stay at tagged releases for dependencies, otherwise stuff could "randomly" break for us with 3rd party upstream changes.